packages/hooks/src/usePhoneVerification.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const usePhoneVerification = () => {
+    const [show_phone_verification_page, set_show_phone_verification_page] = React.useState(true);
+
+    return {
+        show_phone_verification_page,
+        set_show_phone_verification_page,
+    };
+};
```

Comment:
will a context Provider work instead of hook?

Reply:
Removed this hook as we are not using it on the latest code update